presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1876,6 +1877,11 @@ public SystemSessionProperties(
                         "Experimental: When probe side size is unknown but build size is within broadcast limit, choose broadcast join",
                         featuresConfig.isBroadcastJoinWithSmallBuildUnknownProbe(),
                         false),
+                booleanProperty(
+                        PREFER_BROADCAST_JOIN_OVER_PARTITIONED_JOIN,
+                        "When broadcast join is available, choose broadcast join",
+                        featuresConfig.isPreferBroadcastJoinOverPartitionedJoin(),
+                        false),
```

Comment:
Should we set the default to true? Actually I feel there are too many properties for broadcast join already. Adding a new one will be making the cluster management harder. If it's safe to enable it, can we just make it the default behavior?